certbot/tests/error_handler_test.py

```diff
@@ -98,9 +97,9 @@ def test_bad_recovery(self):
                                                **self.init_kwargs)
         bad_func.assert_called_once_with()
 
-    # On Windows, this test kills pytest itself !
-    @test_util.broken_on_windows
     def test_bad_recovery_with_signal(self):
+        if not self.signals:
+            self.skipTest(reason='No signals can be handled by ErrorHandler.')
```

Comment:
Since this string will sometimes be included in test output, can we modify it slightly to say:

> Signals cannot be handled on Windows.

The thinking is just making it clear to the person running the tests why there's this limitation.

Reply:
Fixed